.github/workflows/release_snapshot.yml

```diff
@@ -95,8 +99,10 @@ jobs:
           python -m pip install --user --upgrade pip
           pip install -r requirements.txt
 
-      - run: |
-          python release_creation/main.py \
+      - name:  "Generate Bundle"
```

Comment:
Since we don't have a standard this is your choice, but naming all the steps can be useful when monitoring a workflow.  Does not change how it works - just makes it a bit more reader friendly and easier to maintain for someone else coming in to modify.

Reply:
Good point, added names to pretty much all the steps